### ISSUE TITLE
make installcheck with Postgres 10.1

### ISSUE THREAD

**Issue by a3nm** [PierreSenellart/provsql#4]
Postgres 10.1 is out now. Testing with Debian's postgresql-10 version 10.1-1, "make installcheck" fails on "test add_provenance", probably just because of a formatting change. Here is the diff:

```
*** /home/a3nm/git/provsql/test/expected/add_provenance.out     2016-08-17 14:24:27.733163218 +0200
--- /tmp/tmp.provsqlScjO/results/add_provenance.out     2017-12-17 20:50:32.320742373 +0100
***************
*** 4,18 ****
   
  (1 row)
  
!                                     Table "public.personal"
!      Column     |         Type         |                       Modifiers                       
! ----------------+----------------------+-------------------------------------------------------
!  id             | integer              | not null default nextval('personal_id_seq'::regclass)
!  name           | character varying    | 
!  position       | character varying    | 
!  city           | character varying    | 
!  classification | classification_level | 
!  provsql        | provenance_token     | default uuid_generate_v4()
  Indexes:
      "personal_pkey" PRIMARY KEY, btree (id)
      "personal_provsql_key" UNIQUE CONSTRAINT, btree (provsql)
--- 4,18 ----
   
  (1 row)
  
!                                        Table "public.personal"
!      Column     |         Type         | Collation | Nullable |               Default                
! ----------------+----------------------+-----------+----------+--------------------------------------
!  id             | integer              |           | not null | nextval('personal_id_seq'::regclass)
!  name           | character varying    |           |          | 
!  position       | character varying    |           |          | 
!  city           | character varying    |           |          | 
!  classification | classification_level |           |          | 
!  provsql        | provenance_token     |           |          | uuid_generate_v4()
  Indexes:
      "personal_pkey" PRIMARY KEY, btree (id)
      "personal_provsql_key" UNIQUE CONSTRAINT, btree (provsql)


```

**Comment by PierreSenellart** [comment]
Which version are you using? This was fixed by 948c9c932a770a618d05dde0702ca0e0ee4307f2 released on September 26.

**Comment by a3nm** [comment]
Indeed, with the latest version it works. So if you want you can just update the readme to say that Postgres 10.1 was tested.
